VERSION

```diff
@@ -1 +1 @@
-0.18.0
\ No newline at end of file
+0.19.0
```

Comment:
@ryanking we do releases on our own right?

Reply:
Sorry, I wasn't sure if I was suppose to rev the version or not.

Reply:
Yes, please do not change this file

Reply:
@ryanking it is back to 0.18.0